reference/configuration/swiftmailer.rst

```diff
@@ -303,3 +303,9 @@ Each mailer is registered as a service::
 
     // returns the second mailer
     $container->get('swiftmailer.mailer.second_mailer');
+
+.. caution::
+
+    The configuration in the ``swiftmailer`` key (see above) is a shortcut
+    syntax for the default mailer. If you use multiple mailers, you need to
+    set options like ``disable_delivery`` separately for the additional mailers.
```

Comment:
Thank you, Christian, I think your wording sounds great and explains the issue well.
If i interpret the [configuration code](https://github.com/symfony/swiftmailer-bundle/blob/master/DependencyInjection/SwiftmailerExtension.php) correctly, the default mailer doesn't have to be named `default`, but it has to be configured as `default_mailer`.

What do you think of the following suggestion?

```yaml
.. caution::

    When configuring multiple mailers, options must be placed under the appropriate
    mailer key of the configuration. Options placed directly under the ``swiftmailer``
    key will not be applied to all mailers, but will configure the ``default_mailer`` instead.
```